src/main/app/src/hooks/browseActivitiesFilter.js

```diff
@@ -6,9 +6,18 @@ const filterActivities = (links, activityType, linkFilters, updateFilteredLinks)
     updateFilteredLinks(links);
   else
     updateFilteredLinks(links.filter(link =>
-      activityType === "games" ? filterGames(link, linkFilters) :
-      activityType === "articles" ? filterArticles(link, linkFilters) :
-      filterVideos(link, linkFilters)
+      {
+        switch (activityType) {
+          case "games":
+            filterGames(link, linkFilters);
+            break;
+          case "articles":
+            filterArticles(link, linkFilters);
+            break;
+          default:
```

Comment:
Don't put video as default: if there is a bug in the future that passes the wrong type it would be hidden by this.

Make a case for video and make default an error state.

Reply:
Done. Will change that. Thank you!